src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        return String.valueOf(Color.values()[new Random().nextInt(Color.values().length)]);

```

Comment:
don't use String.valueOf() to get String representation of enum constant, use name() method of enum